src/cf-expandables/src/cf-expandables.less

```diff
@@ -16,186 +16,186 @@
 // $color- variables are from 18F's US Web Design Standards
 // https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
 
-// .expandable
+// .o-expandable
 @expandable-focus:              #046b99; // $color-primary-alt-darkest
 
-// .expandable_label
+// .o-expandable_label
 @expandable_label-text:         #323a45; // $color-gray-dark
 
-// .expandable_link
+// .o-expandable_link
 @expandable_link-text:          #205493; // $color-primary-darker
 
-// .expandable__padded
+// .o-expandable__padded
 @expandable__padded-bg:         #f1f1f1; // $color-gray-lightest
 @expandable__padded-bg-hover:   #d6d7d9; // $color-gray-lighter
 @expandable__padded-divider:    #323a45; // $color-gray-dark
 
-// .expandable-group
+// .o-expandable-group
 @expandable-group_header-text:  @text;
 @expandable-group_header-bg:    #d6d7d9; // $color-gray-lighter
 @expandable-group-bg:           #f1f1f1; // $color-gray-lightest
 @expandable-group-divider:      #323a45; // $color-gray-dark
 
 // Sizing variables
 
-// .expandable_link
+// .o-expandable_link
 @expandable_link-font-size:     14px;
 
 //
 // Recommended expandable pattern
 //
 
-.expandable {
-    margin-bottom: unit(30px / @base-font-size-px, em);
-}
+.o-expandable {
+    position: relative;
+    margin-bottom: unit( 30px / @base-font-size-px, em );
```

Comment:
Realized while pairing expandables aren't always in groups. Let's refactor this later.